microcosm_eventsource/tests/fixtures.py

```diff
@@ -24,6 +24,16 @@
 from microcosm_eventsource.stores import EventStore
 
 
+class FlexibleTaskEventType(EventType):
+    # Used to test is_initial
+    CREATED = event_info(
+        follows=any_of(
+            "CREATED",
+            nothing(),
```

Comment:
There may be an assumption in `microcosm-eventsource` that such transitioning rules couldn't happen - they seem ok to me, since it just means that a particular event can be both initial and non-initial.